backend/battles/tests/test_forms.py

```diff
@@ -196,99 +172,51 @@ def test_cannot_force_a_creator_user(self):
         battle = form.save()
         self.assertEqual(battle.creator, current_user)
 
-    @responses.activate
-    def test_cannot_create_battle_if_pokemon_does_not_exist_in_api(self):
```

Comment:
@filipeximenes, I removed tests like this because when adding a Pokemon in a battle, it comes from the database.